### DISCUSSION TITLE
Please get rid of the white flashing when taking a screenshot

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#219] [Ideas]
Formerly, taking a screenshot caused a short screen blackout, which was tolerable. Now, there is an eye-hurting white flashing. Please remove all flashing effects from the screenshot tool. The file naming window is sufficient visual feedback. Thank you.

Whoever had the idea for this flashing effect probably thought "we must emulate a camera flash for realism! The users will appreciate it!" 

No, we don't.

For now, the only solution is installing `mate-screenshot` and assigning the print screen key to it. Mate-screenshot thankfully has no annoying flashing.

**Comment by billyswong** [comment]
Unless Cinnamon forks `gnome-screenshot`, the situation won't change in any time soon. Somebody filed a patch to make the flashing optional in 2019. And it still sits there not merged by now. The stalemate is likely caused by GNOME abandoned this program and put the screenshot function into the shell starting from GNOME 42.

https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/64
https://release.gnome.org/42/#new-screenshot-experience

**Comment by rcalixte** [comment]
On my machines, I have both [Shutter](https://shutter-project.org/) and [Flameshot](https://flameshot.org/) installed for this purpose. You can re-map the keyboard shortcuts to use either application (Shutter has command-line options for the entire screen vs active window captures and then some. Flameshot is better in some ways for custom edits to screenshots. Shutter only supports X11 sessions for now while Flameshot has Wayland support if you're thinking about that as well.)

Of course, if `mate-screenshot` is working for you, keep using that as long as it does but you do have good options too.

**Comment by goyalyashpal** [comment]
duplicate of 
* #524

**Comment by MinaPoppins** [comment]
I agree, please make the flash optional, it causes me headaches. I am using Flameshot which says it may remove the ability to screenshot without the portal in future updates

**Comment by mtwebster** [comment]
https://github.com/linuxmint/cinnamon/commit/48dc291ef0c7ae485a603bdce650945f146325b3
